content/docs/1.3.0/advanced-resources/os-distro-specific/upgrade-k8s-on-aks.md

```diff
@@ -0,0 +1,58 @@
+---
+title:  Upgrade Kubernetes on Azure AKS
+weight: 8
+---
+
+AKS with Longhorn can be upgraded by node-pool replacement. We cannot use direct `az aks upgrade` because the Longhorn data disk does not get migrated to the new OS image during the AKS upgrade re-image process.
```

Comment:
I thought AKS will automatically do node replacement when they upgrade nodes, so the data will be taken care of by Longhorn replica rebuilding. But, here you need to do it manually by adding another node group then deleting the old one.

Reply:
After @c3y1huang explanation, ask will use in-place node update w/ root disk reimage, so the disk will be not be migrated and Longhorn will not try to recreate it as well, so there are no disks allowing replica rebuilding in updated nodes anymore. 

Leet's just rephase a bit, then all fine.